dependencies.gradle

```diff
@@ -5,12 +5,12 @@
  */
 
 ext {
-    gradlePluginVersion = "4.1.2"
-    kotlinVersion = "1.4.30"
-    compileSdkVersion = 29
-    buildToolsVersion = '29.0.2'
+    gradlePluginVersion = "7.0.2"
+    kotlinVersion = "1.5.0"
+    compileSdkVersion = 31
+    buildToolsVersion = '30.0.3'
     targetSdkVersion = compileSdkVersion
-    minSdkVersion = compileSdkVersion
+    minSdkVersion = 25
```

Comment:
Which components we use require this minSdkVersion? Is it not possible to have it in sync with the SDK, which has minSdkVersion = 21?

Reply:
In the PenEventDrawer - context.getColor(colorResId) requires lev 23
In the DragSourceFragment - view.startDragAndDrop() requires lev 24
In the Shortcuts.kt - ShortcutManager requires lev 25